companies/tasks/report_processing.py

```
@@ -0,0 +1,231 @@
+            content_vector=data["embedding"],
+            submitted_at=data.get("submitted_at"),
+        )
+
+        if success:
+            Report.objects.filter(id=data["report_id"]).update(
+                processing_status="completed",
+                processed_at=timezone.now(),
+            )
+            logger.info(f"OpenSearch 저장 성공: {data['report_id']}")
+
+        return success
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)